website/content/docs/k8s/dns/views/index.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: Consul DNS views for Kubernetes
+description: ->
+    Kubernetes clusters can use the Consul DNS proxy to return service discovery results from the Consul catalog. Learn about how to configure your k8s cluster so that applications can resolve Consul DNS addresses without gossip communication.
+---
+
+# Consul DNS views for Kubernetes
+
+This topic describes how to schedule a dedicated Consul DNS proxy in a Kubernetes Pod so that Kubernetes Services can use Consul DNS addresses. You can use the Consul DNS proxy to enable service discovery across admin partitions in Kubernetes deployments without using agent gossip.
```

Comment:
```suggestion
This topic describes how to schedule a dedicated Consul DNS proxy in a Kubernetes Pod so that applications in Kubernetes can resolve Consul DNS addresses. You can use the Consul DNS proxy to enable service discovery across admin partitions in Kubernetes deployments without needing to deploy Consul client agents.
```

[Services](https://kubernetes.io/docs/concepts/services-networking/service/) are a specific resources within Kubernetes. Technically it is the application that is performing the DNS resolution, so the sentence should accurately reflect that.

The benefits of not using gossip may not be clear to the reader. I think it'd be better to simply state that this feature does require the deployment of client agents.